app/views/queue_items/index.html.haml

```diff
@@ -4,30 +4,32 @@
       %article
         %header
           %h2 My Queue
-        %table.table
-          %thead
-            %tr
-              %th(width="10%") List Order
-              %th(width="30%") Video Title
-              %th(width="10%") Play
-              %th(width="20%") Rating
-              %th(width="15%") Genre
-              %th(width="15%") Remove
-          %tbody
-            - @queue_items.each do |item|
+        = bootstrap_form_tag update_queue_path
```

Comment:
After I defined it, I was having problems because it behaves like a model backed form even though it is not... so the id, position, and rating don't work-- because model backed forms do not accept an array and store it in params, and the second argument has to be formatted as a hash. Is there any way around this? I read on StackOverflow that you can include form_tags in a model backed form so I'm wondering if it's okay to have them mixed in order to leave the array as is or is that bad practice? In any case I am updating my code to use form_tag and forget about the bootstrap_form_tag since it really isn't providing any benefit here.